src/II/PPWCode.Vernacular.NHibernate.II.csproj

```diff
@@ -11,32 +11,18 @@
 
     <!-- Stylecop -->
     <CodeAnalysisRuleSet>..\StyleCop.ruleset</CodeAnalysisRuleSet>
+    <LangVersion>latest</LangVersion>
```

Comment:
Directory.Build.props already contains LangVersion == 7.0. I would remove it here.
Then you will get build errors because 'default' is only supported from C# 7.1.
So, maybe update LangVersion in Directory.Build.props to 7.3 or 8.0?